repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/InspectProcessDuration.java

```diff
@@ -17,18 +17,18 @@
     private EndProcessNotifier endProcessNotifier;
     private InspectBuilds inspectBuilds;
     private InspectJobs inspectJobs;
-    private BuildRunner buildRunner;
+    private DockerPipelineRunner DockerPipelineRunner;
```

Comment:
* field name should never start with upper-case
* in the future there will be different runner so the field name should not be bound to docker, it can be `pipelineRunner`

Reply:
Hi, your suggestion is to only change the field name and still keep the class name as DockerPipelineRunner ? :)